src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -701,3 +706,23 @@ def _load_from_source(cls, source: Union[str, Path], params_override: Optional[D
             end_time=datetime.datetime.fromisoformat(run_info["end_time"]),
             **kwargs,
         )
+
+    def _copy(self, **kwargs):
```

Comment:
why do we need this? And we do, could change the name to be a more specific one, also improve the docstring about why we need it.